### ISSUE TITLE
Custom alias creation error: "Alias creation time is expired, please retry"

### ISSUE THREAD

**Issue by tylernix** [simple-login/app#1712]
I am using the [SimpleLogin API to create a custom alias](https://github.com/simple-login/app/blob/master/docs/api.md#post-apiv3aliascustomnew). I have created an API key. I can call other endpoints successfully, but when I call the `/alias/custom/new` endpoint, it returns this error every time. 

Request
```
POST https://app.simplelogin.io/api/v3/alias/custom/new
{
    "alias_prefix": "my-new-prefix",
    "alias_suffix": "my-domain-I-set-up",
    "mailbox_ids": [520663, 520971],
    "note": "Testing",
    "name": "Test"
}
```

Response
```
412 PRECONDITION FAILED 
{
    "error": "Alias creation time is expired, please retry"
}
```

Retrying doesn't work. I have tried recreating the API key. I have tried different configurations of field names and values. Nothing seems to work. 

Has anyone else seen this issue?

**Comment by Sherlock1979** [comment]
Yes, I am just having the exact same issue. I guess one of the input parameters is not correct but I don't know. The API documentation is a bit too brief and I couldn't find any working examples just yet.

**Comment by Sherlock1979** [comment]
Yes, there are two issues with your request

1. There is no 'alias_suffix'. It is called 'signed_suffix'.
2. The 'signed_suffix' is not "my-domain-I-set-up" as it needs to be signed. First, you need to call GET /api/v4/alias/options endpoint to get a temporary signed version of your domain name. Each /alias/options call will give you a new 'signed_suffix' and I guess there is an expiration (time limit) to use it.
